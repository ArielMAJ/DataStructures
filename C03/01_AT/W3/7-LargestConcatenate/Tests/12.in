7
5 52 57 517 532 569 581