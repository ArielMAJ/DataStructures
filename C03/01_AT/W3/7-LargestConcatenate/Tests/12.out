58157569553252517 